content/about/notes/1.1/index.md

```diff
@@ -50,31 +50,31 @@ Istio 1.1.
 ## Traffic management
 
 - **New [`Sidecar`](/docs/concepts/traffic-management/#sidecars) Resource**. Enables more fine-grained
-  control over the behavior of the sidecar proxies attached to workloads within a namespace. In 
-  particular adds support to limit the set of services a sidecar will send traffic to. This  
+  control over the behavior of the sidecar proxies attached to workloads within a namespace. In
+  particular adds support to limit the set of services a sidecar will send traffic to. This
   reduces the amount of configuration computed and transmitted to
-  the proxy, improving startup time, resource consumption and control-plane scalability. 
+  the proxy, improving startup time, resource consumption and control-plane scalability.
   For large deployments, we recommend adding a sidecar resource per namespace. Controls are also
   provided for ports, protocols and traffic capture for advanced use-cases.
 
 - **Restrict Visibility of Services**. Added the new `exportTo` feature which allows
-  service owners to control which namespaces can reference their services. This feature is 
+  service owners to control which namespaces can reference their services. This feature is
   added to `ServiceEntry`, `VirtualService` and is also supported on a Kubernetes Service via the
-  `networking.istio.io/exportTo` annotation. 
-  
+  `networking.istio.io/exportTo` annotation.
+
 - **Namespace Scoping**. When referring to a `VirtualService` in a Gateway we use DNS-style name matching
   in our configuration model. This can be ambiguous when more than one namespace defines a virtual service
-  for the same host name. To resolve ambiguity it is now possible to explicitly scope these references 
-  by namespace using a syntax of the form **`[{namespace-name}]/{hostname-match}`** in the `hosts` field. 
-  The equivalent capability is also available in `Sidecar` for egress.  
+  for the same host name. To resolve ambiguity it is now possible to explicitly scope these references
+  by namespace using a syntax of the form **`[{namespace-name}]/{hostname-match}`** in the `hosts` field.
+  The equivalent capability is also available in `Sidecar` for egress.
```

Comment:
Is this required now to use namespace/hostname for a VS to refer to a gateway in a different namespace?   I assume only needed when there is ambiguity.